docs/src/docs/quick-start.md

```diff
@@ -104,15 +104,47 @@ You'll see a new entry named the same as your pack command.
 
 ### 6/ Push the ModelKit to a Remote Repository
 
-You're already logged in to your remote repository, so now you can just push. The naming of your ModelKit will need to be the same as what you see in your `kit list` command (REPOSITORY:TAG). You can even copy and paste it. In my case it looks like:
+The naming of your ModelKit will need to be the same as what you see in your `kit list` command (REPOSITORY:TAG). You can even copy and paste it. In my case it looks like:
 
 ```sh
 kit push docker.io/jozubrad/mymodelkit:latest
 ```
 
+### 7/ Run an LLM Locally
+
+If you're using Kit with LLMs you can quickly run the model locally to speed integration, testing, or experimentation.
+
+Create a new directory for your LLM:
+
+```sh
+mkdir devmode
+cd devmode
+```
+
+Now pull and unpack an LLM ModelKit (we have [several](https://github.com/orgs/jozu-ai/packages)), I've chosen Phi3:
+
+```sh
+kit pull ghcr.io/jozu-ai/phi3:3.8b-mini-instruct-4k-q4_K_M
+kit unpack ghcr.io/jozu-ai/phi3:3.8b-mini-instruct-4k-q4_K_M
+```
```

Comment:
You can also unpack directly here, if we want this to be even simpler (i.e. skip pull)